### PR TITLE
[7157] Align Fimento Partner API documentation with Bank Data Insights API documentation

### PR DESCRIPTION
**Pull request by MrJohanEkman**
image

image

Fimento logo and button color is now aligned, can't match the fonts somehow unfortunately

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,17 +6,12 @@
   <meta charset="UTF-8">
   <title>Fimento API</title>
   <link rel="icon" type="image/png" href="./dist/assets/favicon.png">
-  <link rel="stylesheet" type="text/css" href="./dist/css/global.css" />
+  <!-- <link rel="stylesheet" type="text/css" href="./dist/css/global.css" /> -->
```
Comment (Yunodoctor) [inline index.html]: Any reason we can't just remove  this?

File: index.html [hunk 2]
```diff
@@ -6,17 +6,12 @@
...
 
 <body>
-  <header class="fimento-header">
-    <a href="https://www.fimento.com/">
-      <img src="./dist/assets/fimento_logo.svg" />
-    </a>
-  </header>
   <div>
-    <redoc spec-url='./specifications.yml'></redoc>
+    <redoc></redoc>
```
Comment (Yunodoctor) [inline index.html]: What is this? Do we need this empty tags? If we do it should probably just be `<redoc/>` as well

**Comment by Yunodoctor** [comment]
Why can't we match the font? 🤔

File: index.html (hunk 1 above)
Comment (MrJohanEkman) [reply index.html]: I just accidently hid it but I restored it in my last commit

File: index.html (hunk 2 above)
Comment (MrJohanEkman) [reply index.html]: that is true, since now it is invoked through javascript instead of HTML

**Comment by MrJohanEkman** [comment]
> Why can't we match the font? 🤔

Both documentations are using the default font which is Monserrat, but for some reason they look different when shown in the web browser. Even if you inspect with the Chrome devtools it shows the exakt font, size etc...

File: index.html (hunk 2 above)
Comment (MrJohanEkman) [reply index.html]: Actually it still need the html tag so I will change to <redoc/>

**Review by Erik-Nilsson-Fimento — APPROVED** [review]

**Review by xyhuang1116 — APPROVED** [review]

**Pull request merged**